docs/learn/learn-staking.md

```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
 
 import RPC from "./../../components/RPC-Connection"
 
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements NPoS (Nominated
-Proof-of-Stake), a relatively novel and sophisticated mechanism to select the validators who are
-allowed to participate in its [consensus](learn-consensus.md) protocol. The NPoS encourages
-{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} holders to participate as nominators.
+Here you will learn about what staking is, why it is important and how it works on
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}.
+
+## Introduction to Proof-of-Stake (PoS)
+
+Blockchain networks use [consensus](learn-consensus.md/#why-do-we-need-consensus) mechanisms to finalize
+blocks on the chain. Consensus is the process of agreeing on something, in this case, the progression
+of the blockchain or how blocks are added to the chain. Consensus consists of two actions:
+
+- `block production`, i.e. the way multiple blocks candidates are produced, and 
```

Comment:
I would not use code formatting for what is not code.

Reply:
Fixed throughout the page